WORKSPACE.bazel

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A workspace for gapic-showcase
+"""
+workspace(name = "gapic_showcase")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+##
+# required by googleapis
+#
+http_archive(
+  name = "com_google_api_codegen",
+  strip_prefix = "gapic-generator-2.4.0",
+  urls = ["https://github.com/googleapis/gapic-generator/archive/v2.4.0.zip"],
```

Comment:
Why do we need to depend on the monolith, if we're trying to deprecate it?

Reply:
that was me using the wrong flag in the googleapis dependency
fixed